src/app/common-ui/Popup/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styles from './style.scss';
+import PropTypes from 'prop-types';
+
+import CloseButton from '../../../assets/images/close-button.png';
+
+function Popup(props) {
+  const { onClose, title, content, buttonText, variant, onButtonClick } = props;
+  return (
+    <div className={styles.overlay}>
+      <div className={styles.container}>
+        <div className={styles.titleBar}>
+          <div>
+            <div
+              className={`${styles.title} ${
+                variant === 'success' ? styles.titleColorSuccess : styles.titleColorFailed
+              }`}
+            >
+              {title}
+            </div>
+          </div>
+          <div
+            className={styles.closeImgDiv}
+            onClick={variant === 'error' ? onClose : onButtonClick}
+          >
+            <img className={styles.closeImg} src={CloseButton} alt="close" />
+          </div>
+        </div>
+        <div className={styles.content}>
+          <div>{content}</div>
+        </div>
+        <button
+          className={styles.closeBtn}
+          type="submit"
+          onClick={variant === 'error' ? onClose : onButtonClick}
```

Comment:
`onClick={onButtonClick}`

`variant` should only be used to control the UI, not logic.